src/utils/utils.js

```diff
@@ -28,14 +30,14 @@ const createJWTCookie = (user, res, tokenName = 'token') => {
     });
 };
 
-const verifyToken = (token, res, tokenName = 'token') => {
+const verifyToken = (token, res, tokenName = keys.tokenName) => {
     try {
-        const decoded = verify(token, secretkey);
+        const decoded = verify(token, keys.publicKey);
```

Comment:
I think when you verify you need to also mention the algorithm you used to sign for added security, maybe look that up and add that as well